git

```diff
@@ -148,6 +148,10 @@ git diff --no-index path/to/file/A path/to/file/B
 git fetch --all
 git reset --hard origin/master
 
+# To pull a remote branch but rebase any locally differing comments onto the
```

Comment:
"comments" here is a typo for "commits", right?

Reply:
You are correct, not sure how I missed that. Fixed.